allure-selenide/src/main/java/io/qameta/allure/selenide/AllureSelenide.java

```diff
@@ -71,6 +73,11 @@ public AllureSelenide savePageSource(final boolean savePageHtml) {
         return this;
     }
 
+    public AllureSelenide includeSelenideLocators(boolean includeSelenideLocators) {
```

Comment:
just a proposal 
What do you think if rename it to `includeSelenideLogs`?
Not sure, but "Selenide Locators" can be misunderstood...

Reply:
selenideLogs is actually also not right, these are not "logs", perhaps `includeSelenideSteps`?